### DISCUSSION TITLE
Text extraction from image in pdf file

### DISCUSSION THREAD

**Discussion by DorotaJanosz** [pypdfium2-team/pypdfium2#318] [General]
Hi,

I have made a screenshot and saved it as a pdf file. I want to extract text from that pdf but I am getting empty output. I know that such file does not have a text layer and it can by added by some OCR tool like ocrmypdf, but I am searching for any function within pypdfium2 which would do that internally. Could you please advice me if there is such method? If not, could you advice on any other workaround solution?

**Comment by mara004** [comment]
No, what you need is OCR, and that is not pdfium's job.
ocrmypdf is a great choice for this task.
